website/src/components/CustomCarousel.tsx

```diff
@@ -19,16 +19,16 @@ interface CarouselProps {
 
 interface CarouselItem {
     id: number
-    backgroundClass: string
+    backgroundClass?: string
     buttonLabel: string
     headerClass: string
     text: string | ReactNode
     itemClass: string
 }
 
 const CustomCarousel: FunctionComponent<CarouselProps> = props => {
-    const carouselMainStyles = 'd-flex flex-wrap'
-    const carouselRightPanelStyles = 'col-lg-6 col-md-8 col-sm-12 mt-lg-5 ml-md-6 px-lg-0'
+    const carouselMainStyles = 'd-flex flex-wrap flex-lg-row flex-column'
```

Comment:
I noticed that the carousel arrows push the page layout horizontally from the `sm` to `md` breakpoints. You might need to modify this a bit.

![Screen Shot 2022-03-16 at 2 50 41 PM](https://user-images.githubusercontent.com/1733936/158665219-62fcfeac-138b-412f-8b83-4709d57e6779.png)

Reply:
Weird. Why am I not seeing that?
![Screen Shot 2022-03-17 at 3 58 28 PM](https://user-images.githubusercontent.com/43454804/158885516-c624443c-e98f-42d2-8428-81da37eefa0a.png)

Reply:
Weird, I'm not sure 🤷🏻‍♂️  I thought I was cached but I hard refreshed and also tried in incognito and I can still reproduce.  Check out [this loom](https://www.loom.com/share/fd751290063d4fddaeb4d20520a590a1)

Reply:
I ended up updating those styles for other reasons, so marking this as resolved for now.